SPECS/libhugetlbfs/libhugetlbfs.spec

```
@@ -0,0 +1,366 @@
+# RHBZ#1628794 undersized SHMMAX when running on aarch64
+# https://github.com/libhugetlbfs/libhugetlbfs/issues/39
+Patch3:         0003-tests_shm-perms_adjust_max_segment_size_for_bigger_hugepages.patch
+# Downstream patches to remove an IA-64 target leftover that breaks the
+# tests install and fix run_tests.py path for hugeadm tool call
+Patch4:         0004-tests-makefile-fix.patch
+Patch5:         0005-tests-run_tests-fix-hugeadm-path.patch
+# Patch7: huge_page_setup_helper.py Python3 conversion
+# Upstream tickets:
+# Fedora: https://bugzilla.redhat.com/show_bug.cgi?id=1598570
+# libhugetlbfs: https://github.com/libhugetlbfs/libhugetlbfs/issues/35
+Patch6:         0006-huge_page_setup_helper-python3-convert.patch
+# Fixes for downstream COVSCAN and RPMDiff execshield complaints:
+Patch7:         0007-tests-fix-covscan-SHELLCHECK_WARNING-complaints.patch
+Patch8:         0008-tests-include-missing-LDFLAGS-to-make-targets.patch
+# __morecore feature has been removed from glibc 2.34+ so disable it
+Patch9:         0009-Disable-hugepage-backed-malloc-if-__morecore-is-not-.patch
+%global _hardened_build 1
+%define ldscriptdir %{_datadir}/%{name}/ldscripts
+BuildRequires:  execstack
+BuildRequires:  glibc-devel
+BuildRequires:  python3-devel
+BuildRequires:  which
+
+%description
+libhugetlbfs is a library which provides easy access to huge pages of memory.
+It is a wrapper for the hugetlbfs file system. Applications can use huge pages
+to fulfill malloc() requests without being recompiled by using LD_PRELOAD.
+Alternatively, applications can be linked against libhugetlbfs without source
+modifications to load BSS or BSS, data, and text segments into large pages.
+
+%package devel
+Summary:        Header files for libhugetlbfs
+Group:          Development/Libraries
+Requires:       %{name} = %{version}-%{release}
+
+%description devel
+Contains header files for building with libhugetlbfs.
+
+%package utils
+Summary:        Userspace utilities for configuring the hugepage environment
+Group:          Applications/System
+Requires:       %{name} = %{version}-%{release}
+Requires:       perl
+Requires:       python3
+
+%description utils
+This packages contains a number of utilities that will help administrate the
+use of huge pages on your system.  hugeedit modifies binaries to set default
+segment remapping behavior. hugectl sets environment variables for using huge
+pages and then execs the target program. hugeadm gives easy access to huge page
+pool size control. pagesize lists page sizes available on the machine.
+
+%package tests
+Summary:        Test cases to help on validating the library environment
+Group:          Development/Libraries
+Requires:       %{name}-utils = %{version}-%{release}
+
+%description tests
+This packages contains a number of testcases that will help developers
+to verify the libhugetlbfs functionality and validate the library.
+
+%prep
+%setup -q
+%autosetup -p1
```

We only need the second line here:

```suggestion
%autosetup -p1
```